### ISSUE TITLE
BREAKING: Get-MgSite Broken by 1.9.3 with addition of -Path parameter

### ISSUE THREAD

**Issue by JustinGrote** [microsoftgraph/msgraph-sdk-powershell#1237]
Looks like maybe the autogeneration code is now picking up a getByPath method and overwriting the default cmdlet

https://github.com/microsoftgraph/msgraph-sdk-powershell/blob/34b6a456532877d22dfe480b4f983f4aa6d8eb9a/samples/6-Sites.ps1#L4
![image](https://user-images.githubusercontent.com/15258962/166076223-50bf98b4-e0c1-4f0a-85ce-2e8b627bdaeb.png)

`https://graph.microsoft.com/v1.0/sites/root/microsoft.graph.getByPath(path='test')`

# Workaround
Revert to 1.9.2

**Comment by JustinGrote** [comment]
@peombwa Regression warning...

**Comment by peombwa** [comment]
Sorry about this. The bug was introduced when the workload added `sites/{site-id}/getByPath` API to v1.0 endpoint. A [fix](https://github.com/microsoftgraph/msgraph-sdk-powershell/pull/1229) has already been provided, and the patch should be out shortly once https://github.com/microsoftgraph/msgraph-sdk-powershell/pull/1242 is approved.

A workaround is to revert to 1.9.2:
``` powershell
Get-Module 'Microsoft.Graph*' -ListAvailable | where { $_.Version -gt "1.9.2" } | Uninstall-Module -Force
```

**Comment by peombwa** [comment]
We've just released version 1.9.6 with a fix to this issue. Please use `Update-Module Microsoft.Graph` to upgrade to the latest version.
